pkg/state/state_test.go

```diff
@@ -1813,7 +1813,7 @@ func TestHelmState_DiffReleasesCleanup(t *testing.T) {
 				return nil
 			}
 			state = injectFs(state, testfs)
-			if _, errs := state.DiffReleases(tt.helm, []string{}, 1, false, false, []string{}, false, false, false, false); len(errs) > 0 {
+			if _, errs := state.DiffReleases(tt.helm, []string{}, 1, false, false, []string{}, false, false, false, false, false); len(errs) > 0 {
```

Comment:
can you add a testcase when nohooks is true.

Reply:
I've added another test case for the diff test in [8ff82c2](https://github.com/helmfile/helmfile/pull/279/commits/8ff82c2e7c2bf7ca80e9b9cb31727e9ddf102adf).
I also discovered that currently, it's very tedious and tiresome to add new test cases to the `TestApply` series. The process should ideally be a matter of adding a case in go code like 8ff82c2 and the test framework does the rest, of capturing the helmfile stdout output in the first run and use it to verify the helmfile stdout output matches the snapshot for subsequent runs.
It's already streamlined in that way for app_diff_test.go, app_lint_test.go and some others, but not for app_apply_test.go... We'd need to enhance it in another PR!

Reply:
ok. @mumoshu